utility.lua

```diff
@@ -1015,20 +1014,19 @@ function Auxiliary.AddFusionProcMix(c,sub,insf,...)
 					return false
 			end
 			for _,fcode in ipairs(val[i]) do
-				if type(fcode)~='function' then table.insert(mat,fcode) end
+				if type(fcode)~='function' and not mat[fcode] then mat[fcode]=true end
 			end
 		else
 			fun[i]=function(c,fc,sub) return c:IsFusionCode(val[i]) or (sub and c:CheckFusionSubstitute(fc)) end
-			table.insert(mat,val[i])
+			if not mat[val[i]] then mat[val[i]]=true end
```

Comment:
I think `if not ...` is unnecessary

Reply:
https://github.com/Fluorohydride/ygopro-scripts/pull/1271/commits/62e709b70a275404e5e11c48c298569c8756cd18 updated.